test/test_env.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+export GRIST_SESSION_COOKIE="grist_test_cookie"
+export LANGUAGE="en_US"
+export SE_BROWSER="chrome"
+export SE_BROWSER_VERSION="127"
+export SE_DRIVER="chrome-driver"
+export SE_DRIVER_VERSION="127.0.6533.119"
+export TEST_CLEAN_DATABASE="true"
+export TEST_SUPPORT_API_KEY="api_key_for_support"
+
+exec "$@"
```

Comment:
I like the cleanup to have this all in one place, and I'm happy too about pinning the Chrome version.

As a follow-up, we should have some automation to increase the version number with new releases and automatically merge in a new version number if the CI passes.